### PR TITLE
[PSMDB-155] fix the order of paramters laoding

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Comment by tplavcic** [comment]
LGTM

File: percona-packaging/debian/percona-server-mongodb-34-server.mongod.init [hunk 1]
```diff
@@ -128,8 +128,11 @@ start_server() {
     ulimit -u 64000
   fi
 
-# Start the process using the wrapper
   /usr/bin/percona-server-mongodb-helper.sh
+  if [ -f /etc/default/$NAME ] ; then
```
Comment (denis-protivenskii) [inline percona-packaging/debian/percona-server-mongodb-34-server.mongod.init]: Let's be pedantic and use screening for filenames.
"/etc/default/$NAME"

**Comment by denis-protivenskii** [comment]
@tplavcic revisit once again?

**Comment by tplavcic** [comment]
LGTM

**Pull request merged**